tests/test_state.py

```diff
@@ -7,6 +7,7 @@
 
 
 def test_convm():
+    """Set up a convoluston matrix."""
```

Comment:
Functional test for the...

Reply:
spelling correction: "convolution".

Functional test for the Convolution matrix.